src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ChefExtensionTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Compute.Test.ScenarioTests
+{
+    public class ChefExtensionTests
+    {
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestChefExtensionBasic()
+        {
+            ComputeTestController.NewInstance.RunPsTest("Test-SetChefExtensionBasic");
```

Comment:
@markcowl , @shahabhijeet the library related issues got resolved. I am facing an issue at this line now. 
```
The term 'Test-SetChefExtensionBasic' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```
I am missing some link. How is `Test-SetChefExtensionBasic` made available here? Do I have to add some path to the PS file where this method is defined?
I am referring `DiagnosticExtensionTests`, but I couldn't find any reference for `Test-DiagnosticsExtensionBasic` method or `DiagnosticExtensionTests.ps1` file added anywhere.